src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -990,16 +990,25 @@ int publish_commit(rgw::sal::Object* obj,
         const auto ret = push_endpoint->send_to_completion_async(
 	  dpp->get_cct(), event_entry.event, res.yield);
         if (ret < 0) {
-          ldpp_dout(dpp, 1) << "ERROR: push to endpoint "
-			    << topic.cfg.dest.push_endpoint
-			    << " failed. error: " << ret << dendl;
+          ldpp_dout(dpp, 1)
+              << "ERROR: push to endpoint " << topic.cfg.dest.push_endpoint
+              << " bucket: " << event_entry.event.bucket_name
```

Comment:
These log lines (lines 995-998 and 1004-1011) have whitespace both at the beginning and at the end - leading to having 2 spaces between words. I think we can get rid of one of them at these log lines.